wayland.c

```diff
@@ -376,11 +376,16 @@ static void layer_surface_handle_configure(void *data,
 		uint32_t serial, uint32_t width, uint32_t height) {
 	struct mako_surface *msurface = data;
 
+	zwlr_layer_surface_v1_ack_configure(surface, serial);
```

Comment:
`ack_configure` must always be followed by `commit`. If the old buffer is suitable, we can perform a commit without attaching a new buffer.

Reply:
I tried to add a wl_surface_commit.

Reply:
Yup, sounds good!